packages/grid-pro/src/vaadin-grid-pro-inline-editing-mixin.js

```diff
@@ -160,6 +160,71 @@ export const InlineEditingMixin = (superClass) =>
       }
     }
 
+    /**
+     * Override grid method to set tabindex and role
+     * on the cell content element for edit columns.
+     *
+     * @param {HTMLElement} cell
+     * @param {GridColumn} column
+     * @protected
+     * @override
+     */
+    _configureBodyCell(cell, column) {
+      if (this._isEditColumn(column)) {
+        cell.removeAttribute('tabindex');
+        cell._content.setAttribute('tabindex', '-1');
```

Comment:
While the `tabindex` is now initially defined on the cell content, clicking on a cell still modifies the `tabindex` of the cell element:

https://user-images.githubusercontent.com/357820/177135545-6d45be07-5a06-4d1f-99f9-8571bc21798f.mov

Same for moving the focus with the keyboard.

Reply:
Thanks, now I see that there are several more places where focus is set on the cell. I will update those.